test/cli/daemon.js

```diff
@@ -65,7 +65,7 @@ describe('daemon', () => {
   let ipfs
 
   beforeEach(() => {
-    repoPath = '/tmp/ipfs-test-not-found-' + Math.random().toString().substring(2, 8)
+    repoPath = path.join(os.tmpdir(), 'ipfs-test-not-found-' + Math.random().toString().substring(2, 8))
```

Comment:
If you don't mind here as well (ref https://github.com/ipfs/ipfs-performance-profiling/pull/12#discussion_r154862400)